New Bank/NewBank/newbank/server/NewBankClientHandler.java

```diff
@@ -50,6 +44,10 @@ public void run() {
 					System.out.println("Request from " + customer.getKey());
 					String responce = bank.processRequest(customer, request);
 					out.println(responce);
+					if (responce.equals("LOGOFF")) {
+						loggedIn = false;
+						run();
+					}
```

Comment:
Added in this condition to catch the LOGOFF and restart the run() from the beginning.